zlp-scheduler/db/seeds.rb

```diff
@@ -15,11 +38,118 @@
         @user.password = "Temp"
         @user.save
         
+@cohort = Cohort.new
+        @cohort.name = "Test Cohort"
+        @cohort.term_id = 202031
```

Comment:
Is this creating a cohort that is attached to the current term? If so, why not use `@term.id`?